### DISCUSSION TITLE
Import and Export Settings

### DISCUSSION THREAD

**Discussion by markbongolan** [cpyarger/obs-midi#121] [Ideas]
Hello! Thanks for your good work here, really helpful. :) 

By the way, can we include an import and export option so that I can create different setups for different scene collections? Thanks!

**Comment by cpyarger** [comment]
You already can do that, each scene collection and profile gets its own
file.

**Reply by markbongolan** [reply]
Thank you! Where can I see that? Everytime I close a scene collection/obs my mappings are gone

**Comment by markbongolan** [comment]
Hello! Where can I possibly see that?

ᐧ

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

On Wed, Mar 24, 2021 at 2:48 PM Christopher P. Yarger <
***@***.***> wrote:

> You already can do that, each scene collection and profile gets its own
> file.
> Warmest Regards,
>
>
> Christopher P. Yarger
>
> Phone: 802-505-7574
> Skype: cpyarger
>
> /************************************
> Based upon email reliability concerns,
> please send an acknowledgment in response to this note.
>
>
> On Wed, Mar 24, 2021 at 12:09 AM markbongolan ***@***.***>
> wrote:
>
> > Hello! Thanks for your good work here, really helpful. :)
> >
> > By the way, can we include an import and export option so that I can
> > create different setups for different scene collections? Thanks!
> >
> > —
> > You are receiving this because you are subscribed to this thread.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/cpyarger/obs-midi/discussions/121>, or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AANSE6UJIKTZ4M7BDOYU54TTFFQY5ANCNFSM4ZWOXPHQ
> >
> > .
> >
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cpyarger/obs-midi/discussions/121#discussioncomment-521399>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ATMDO6SVPYLRQWUROTTULI3TFGDLTANCNFSM4ZWOXPHQ>
> .
>


-- 
Regards,

Mark Jopette Bryle S. Bongolan

**Comment by cpyarger** [comment]
The files are in the obs plugin_config folder. It depends on your operating system where that is.  I don't really recommend poking the config files directly though. It's an easy way to crash obs

**Reply by markbongolan** [reply]
I can't seem to find it. After switching scenes and turning off my computer, all my programmed mappings are gone.

**Reply by markbongolan** [reply]
![image](https://user-images.githubusercontent.com/81278842/112823270-a6ffd400-90bb-11eb-9df1-64dd8ef8c7cb.png)


is this the plugin_config folder? Scenes with mapping that I did have .json.bkp file but all my mappings are gone.

**Reply by cpyarger** [reply]
yes this is the plugin config folder.

**Comment by cpyarger** [comment]
I wont be adding a import/export function directly though. as stated. they are only good for the specific scene collection and profile that you have set up in obs
